cmd/draft/init.go

```diff
@@ -45,15 +49,21 @@ func newInitCmd(out io.Writer, in io.Reader) *cobra.Command {
 
 	f := cmd.Flags()
 	f.BoolVar(&i.dryRun, "dry-run", false, "go through all the steps without actually installing anything. Mostly used along with --debug for debugging purposes.")
+	f.StringVarP(&i.configFile, "config", "f", "", "specify default plugins and pack repositories in a TOML file")
 
 	return cmd
 }
 
 // runInit initializes local config and installs Draft to Kubernetes Cluster
 func (i *initCmd) run() error {
 
+	plugins, repos, err := parseConfigFile(i.configFile)
```

Comment:
is it still possible to run `draft init` without this file present?

Reply:
oh goodness. I'll fix that first before merging. good catch!!